src/psbt/mod.rs

```diff
@@ -1290,6 +1290,9 @@ where
         tr_derived.iter_scripts().zip(tr_xpk.iter_scripts())
     {
         debug_assert_eq!(_depth_der, depth);
+        let ms_derived = ms_derived.as_miniscript().unwrap();
+        let ms = ms.as_miniscript().unwrap();
+
```

Comment:
In 899269acefeb534f140cfd4d7f86b66ad567b55f:

This one we should return an error on. I think we're a long way away from supporting PSBT/PSET with Simplicity policies. Though we should also add a FIXME.

Reply:
Nope, fixed later :)